### ISSUE TITLE
Positional argument % not recognized correctly in post condition.

### ISSUE THREAD

**Issue by pithyless** [borkdude/clj-kondo#315]
**version**

clj-kondo v2019.06.29-alpha

**platform**

MacOS (via homebrew)

**problem**

The `%` is not correctly recognized as valid argument in `:post` condition.

**repro**

```
echo '(defn str-to-str [s]
  {:pre  [(string? s)]
   :post [(string? %)]}
  s)' | clj-kondo --lint -

<stdin>:3:20: info: unresolved symbol %
```

**expected behavior**

Do not report `%` as unresolved symbol in `:post`.

**actual behavior**

`%` reported as unresolved symbol in `:post`.

**Comment by borkdude** [comment]
@pithyless This one should work: 

https://4283-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.06.30-alpha-SNAPSHOT-macos-amd64.zip

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/09f1cde1d021646cf7ecc09e31679834ed2f959e

**Comment by pithyless** [comment]
> @pithyless This one should work:
> 
> https://4283-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.06.30-alpha-SNAPSHOT-macos-amd64.zip

Confirming this binary fixes issue.

**Comment by borkdude** [comment]
Thanks for testing. I hope to release a new version coming weekend.
